### ISSUE TITLE
508-defect-2 [KEYBOARD, COGNITION]: Delete or hide empty link within VA trust percentages

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23834]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

There is an empty link after the VA trust percentages which is throwing an axe issue (links must have discernible text). This may throw keyboard and screen readers off due to the extra empty tab stop.

## Acceptance Criteria

- [ ] Empty link is removed or hidden from AT on the page

## Steps to Recreate

Run an axe scan and confirm that an issue is flagged for "links must have discernible text"

## Proposed Solution 

Delete the link or add `hidden="true"` as an attribute.

## Screenshots or Trace Logs
<img width="1295" alt="Screen Shot 2021-04-27 at 9 44 07 AM" src="https://user-images.githubusercontent.com/14154792/116261923-1c57d680-a746-11eb-915c-d9b88aeebf11.png">

**Comment by brianalloyd** [comment]
This might be something that needs to be conditionally rendered, creating ticket that will address this in totality for CLP.

**Comment by laflannery** [comment]
I do not believe this is relevant anymore, I cannot find the screenshotted area on the page.  Will close in 1 week unless told otherwise
